docs/skopeo.1.md

```diff
@@ -0,0 +1,191 @@
+  By default, uses the authorization state in `~/.docker/config.json`,
+  which is set e.g. using `(docker login)`.
+
+  **oci:**_path:tag_
+  An image _tag_ in a directory compliant with “Open Container Image
+  Layout Specification” at _path_.
+
+# OPTIONS
+
+  _--debug_ enable debug output
+
+  _--username username_ for accessing  the registry
+
+  _--password password_ for accessing the registry
+
+  _--cert-path path_ Use certificates at _path_ (cert.pem, key.pem) to connect to the registry
+
+  _--tls-verify bool-value_ Verify certificates
+
+  _--help, -h_ Show help
+
+  _--version, -v_ print the version number
+
+# COMMANDS
+
+## skopeo copy _--sign-by=key-ID source-image destination-image_
+Copy an image (manifest, filesystem layers, signatures) from one location to another.
+
+  `source-image`
+
+  `destination-image` use the "image name" format described above
+
+  _--sign-by=key-id_ add a signature using that key ID for an image name corresponding to `destination-image`
+
+Existing signatures, if any, are preserved as well.
+
+## skopeo delete _image-name_
+Mark _image-name_ for deletion.  To release the allocated disk space, you need to execute the docker registry garabage collector. E.g.,
+
+```sh
+docker exec -it registry bin/registry garbage-collect /etc/docker/registry/config.yml
+```
+
+Additionally, the registry must allow deletions by setting `REGISTRY_STORAGE_DELETE_ENABLED=true` for the registry daemon.
+
+## skopeo inspect _[--raw] image-name_
+Return low-level information about _image-name_ in a registry
+
+  _--raw_ output raw manifest, default is to format in JSON
+
+  `image-name` name of image to retrieve information about
+
+## skopeo layers _image-name_
+Get image layers of _image-name_
+
+  `image-name` name of the image to retrieve layers
+
+## skopeo standalone-sign _manifest docker-reference key-fingerprint --output|-o signature_
+This is primarily a debugging tool, or useful for special cases,
+and usually should not be a part of your normal operational workflow; use `skopeo copy --sign-by` instead to publish and sign an image in one step.
+
+  `manifest` Path to a file containing the image manifest
+
+  `docker-reference` A docker reference to identify the image with
+
+  `key-fingerprint` Key identity to use for signing
+
+  _--output, -o_ output file
+
+## skopeo standalone-verify _manifest docker-reference key-fingerprint signature_
+Verify a signature using local files, digest will be printed on success.
+
+  `manifest` Path to a file containing the image manifest
+
+  `docker-reference` A docker reference expected to identify the image in the signature
+
+  `key-fingerprint` Expected identity of the signing key
+
+  `signature` Path to signature file
+
+**Note:** If you do use this, make sure that the image can not be changed at the source location between the times of its verification and use.
+
+## skopeo help
+show help for `skopeo`
+
+# EXAMPLES
```

Comment:
Great.

The examples are currently inconsistent WRT leading `#`, and it would probably be better to drop it, the `sh` markup parser seems to treat them as comment indicators and makes the commands (the most important part) grey.